packages/playwright-core/src/server/debugger.ts

```diff
@@ -63,6 +59,15 @@ export class Debugger extends EventEmitter implements InstrumentationListener {
       await this.pause(sdkObject, metadata);
   }
 
+  async _doSlowMo() {
+    await new Promise(f => setTimeout(f, this._slowMo));
```

Comment:
Let's fast return when `slowMo` is zero? I am afraid of unnecessary slow downs.

Reply:
It happens on line 67 new code.